package.json

```diff
@@ -23,7 +23,7 @@
     "cssnano": "^3.10.0",
     "express": "^4.16.3",
     "gulp": "^3.9.1",
-    "gulp-better-rollup": "^3.3.0",
+    "gulp-better-rollup": "3.1.0",
```

Comment:
Can we write a test that checks that this is always 3.1.0? Worried this is really easy to make a mistake on.